scripts/eurostat/health_determinants/common/download_input_files.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script calls the download script in the common folder of eurostat,
+the download script takes INPUT_URLs and current directory as input
+and downloads the files.
+"""
+import os
+import sys
+
+from absl import app, flags
+
+# pylint: disable=import-error
+# pylint: disable=wrong-import-position
+# For import common.download
+
+_COMMON_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.insert(1, _COMMON_PATH)
+from common import import_download_details, download
+
+# pylint: enable=import-error
+# pylint: enable=wrong-import-position
+_FLAGS = flags.FLAGS
+flags.DEFINE_string("import_name", "alcohol_consumption",
+                    "Import name for which input files will be downloaded")
+
+
```

Comment:
if the function download_file() is only used here, it can be brought into this script itself.